hazelcast/src/main/java/com/hazelcast/logging/LoggingService.java

```diff
@@ -18,12 +18,8 @@
 
 import java.util.logging.Level;
 
-public interface LoggingService {
+public interface LoggingService extends LoggerFactory {
```

Comment:
In my mental model, `LoggerFactory` abstracts away the logger implementation, so I wouldn't mix `LoggingService` in `LoggerFactory` hierarchy, but that's just my interpretation. wdyt?